src/include/miopen/find_solution.hpp

```diff
@@ -97,6 +97,11 @@ auto FindSolutionImpl(rank<1>, Solver s, const Context& context, Db& db)
         }
     }
 
+    if(miopen::FindMode(context).IsNtHybrid() && !s.IsEnabledForNtHybrid())
+    {
+        // For now, let's use this for solutions skipped due to NT_HYBRID.
+        return {static_cast<miopenStatus_t>(100)};
```

Comment:
Don't we need to call `GetPerformanceConfig` to find out if there are tunable parameters?

Reply:
Ah wait, if a solver is not tunable then it won't have a `GetPerformanceConfig` method. So this should just be:

```cpp
if(miopen::FindMode(context).IsNtHybrid()
    return {static_cast<miopenStatus_t>(100)};
```

Reply:
Yes. And `FindSolutionImpl(rank<1>,...)` is called with tunable solvers only.